onos-topo/templates/deployment.yaml

```diff
@@ -50,11 +50,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-            - name: RSA_PUBLIC_KEY
-              valueFrom:
-                secretKeyRef:
-                  name: {{ template "onos-topo.fullname" . }}-jwtkey
-                  key: rsa_public_key.pem
+            - name: OIDC_SERVER_URL
```

Comment:
Do I need to change anything in the library function?

Reply:
i made a big change in lib function
https://github.com/onosproject/onos-lib-go/pull/56